_component_code/footer.md

```diff
@@ -9,3 +9,57 @@ collection: component_code
 ---
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
+
```

Comment:
@technabors the only thing we may need is the image for here. See the image on the overview here: https://jira.rax.io/browse/SURF-62

Reply:
I'm really not a fan of the existing image. I'll make a new one in the same abstracted thumbnail style Ty created and upload it to the JIRA

Reply:
Thanks!